README.md

```diff
@@ -139,6 +140,18 @@ Table of Contents
 
 **[⬆ back to top](#table-of-contents)**
 
+## Cloud management solutions
+  * [Brainboard](https://www.brainboard.co) - visually build and manage cloud infrastructures from end-to-end:
```

Comment:
We do not add single entry catagories, either add this to an existing category - as one line - or move other matching items into this category

Reply:
I moved Terraform.io into this category as well. Let me know please if it's OK.

Reply:
Please make a one line description. Giant multi offering services like GitHub or huge clouds get a multi line. The rest just isn’t that varied.

Reply:
Done.